clients/cpp-console/src/common/mainAudio.cpp

```diff
@@ -66,26 +84,54 @@ int main(int argc, char** argv)
     string configFilePath = argv[1];
     string s;
     const char * device = "default";
-    bool keywordListeningEnabled = false;
+    KeywordActivationState keywordActivationState = KeywordActivationState::Undefined;
     bool volumeOn = false;
     
     IAudioPlayer* player;
     shared_ptr<AgentConfiguration> agentConfig;
     shared_ptr<DialogServiceConnector> dialogServiceConnector;
     
-    auto startKwsIfApplicable = [&]()
+    auto StartKws = [&]()
     {
-        log_t("startKWS called");
-        if (keywordListeningEnabled)
+        log_t("startKwsIfApplicable called");
```

Comment:
looks like you have the wrong function name here

Reply:
Good catch. Fixing now.